### PR TITLE
the most fun post I've written this year. Also fixes a typo in the la…

### PR DESCRIPTION
**Pull request by serverless-mom**
…st blog post as a drive-by, which is a terrible habit but... :shrug:

### CODE REVIEW

File: blog/2023-09-16-ten-reasons-not-add-observability.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+Downscaling? resource overuse? You call it a wasted Operations budget, I call it stimulating the economy. Observability might be a great way to identify possible efficiencies, and even whole clusters that nobody is using, but really who needs the bother.
+
+When the company has to massively reduce size due to budget overruns, *then* we can downscale whole departments at a time. And without observability, that should happen in no time!
+
+How else will Jeff Bezos afford another 10-and-a-half minutes in the upper atmosphere?
+
+<figure data-zoomable align='center'>
+    <img src="/img/blog/2023/09/ten_reasons/ten-reasons-4.webp" alt="By G. Lamar - The Upper Atmosphere at 10,000 Feet, CC BY 2.0 - an image of the upper atmosphere"/>
+    <figcaption><i>Nothing more fun than hearing a billionaire refer to your heard-earned infrastructure money as his ‘winnings’!</i></figcaption>
```
Comment (ankit01-oss) [inline blog/2023-09-16-ten-reasons-not-add-observability.md]:
```suggestion
    <figcaption><i>Nothing more fun than hearing a billionaire refer to your hard-earned infrastructure money as his ‘winnings’!</i></figcaption>
```

File: blog/2023-09-16-ten-reasons-not-add-observability.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+<figure data-zoomable align='center'>
+    <img src="/img/blog/2023/09/ten_reasons/ten-reasons-7.webp" alt="a black box"/>
+    <figcaption><i>This is what peak operations performance looks like</i></figcaption>
+</figure>
+
+### 8. What’s the worst that could happen?
+
+Tech debt leads to greater tech debt, but as long as your company’s balance sheets look good, fundamentally things must be okay. Right? Right. After all, as long as operations are working right now, albeit at 99% capacity, what’s the worst that could happen? It’s not like even a tiny spike could shut down your system, a lack of observability could make the problem impossible to diagnose, [and a quarter million people could miss Christmas as a result](https://www.forbes.com/sites/noahbarsky/2023/01/06/southwest-airlines-crisis-exposes-3-reasons-why-great-companies-stumble/?sh=35b373e3481e).
```
Comment (ankit01-oss) [inline blog/2023-09-16-ten-reasons-not-add-observability.md]:
```suggestion
Tech debt leads to greater tech debt, but as long as your company’s balance sheets look good, fundamentally things must be okay. Right? Right. After all, as long as operations are working right now, albeit at 99% capacity, what’s the worst that could happen? It’s not like even a tiny spike could shut down your system, a lack of observability could make the problem impossible to diagnose, <a href = "https://www.forbes.com/sites/noahbarsky/2023/01/06/southwest-airlines-crisis-exposes-3-reasons-why-great-companies-stumble/?sh=35b373e3481e" rel="noopener noreferrer nofollow" target="_blank" >and a quarter million people could miss Christmas as a result</a>.
```

Might have missed telling you about this - for any external link we use the above format. It's a good SEO practice to  give external links `noopener noreferrer nofollow` tags.

**Review by ankit01-oss — APPROVED** [review]
@serverless-mom looks good to me, especially the 10th reason :) Before you merge this just update the date to latest so that it comes up on the top.

File: blog/2023-09-16-ten-reasons-not-add-observability.md (hunk 2 above)
Comment (serverless-mom) [reply blog/2023-09-16-ten-reasons-not-add-observability.md]: for the record you absolutely did tell me this, just missed this link :)

**Pull request merged**